javascriptv3/example_code/medical-imaging/actions/copy-image-set.js

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {fileURLToPath} from "url";
```

Comment:
More formatting issues here, but should be resolved by the aforementioned `prettier` command.